Userland/Libraries/LibWeb/HTML/Navigable.cpp

```diff
@@ -1065,9 +1065,10 @@ WebIDL::ExceptionOr<void> Navigable::populate_session_history_entry_document(
         if (document_resource.has<String>()) {
             navigation_params = TRY(create_navigation_params_from_a_srcdoc_resource(entry, this, target_snapshot_params, navigation_id));
         }
-        // 2. Otherwise, if both of the following are true:
+        // 2. Otherwise, if all of the following are true:
         //    - entry's URL's scheme is a fetch scheme; and
         //    - documentResource is null, or allowPOST is true and documentResource's request body is not failure (FIXME: check if request body is not failure)
+        // then set navigationParas to the result of creating navigation params by fetching given entry, navigable, sourceSnapshotParams, targetSnapshotParams, cspNavigationType, navigationId, and navTimingType.
```

Comment:
is "navigationParas" your typo or Domenic's in the spec? 👀

Reply:
Oops, that's my typo- fixed.